recipes/pyphen/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "Pyphen" %}
+{% set version = "0.9.4" %}
+{% set sha256sum = "abfa9a0ab055341f6e250c1a6bef395c3a06f0e4cba216eeef37f617b32c0bd7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256sum }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - pyphen
+
+about:
+  home: http://pyphen.org/
+  license: LGPLv2+
```

Comment:
Re-checked and this package does include the `COPYING` file, although it's tri-licensed. Fixed.